state/postgresql/postgresdbaccess.go

```diff
@@ -93,14 +93,15 @@ func (p *postgresDBAccess) setValue(req *state.SetRequest) error {
 		return fmt.Errorf("missing key in set operation")
 	}
 
-	var valueBytes []byte
-
 	// Convert to json string
-	valueBytes, err = json.Marshal(req.Value)
-	if err != nil {
-		return err
+	var bt []byte
```

Comment:
if this code is used in multiple places, it would be great to create shared util.